app/src/main/java/unipd/se18/ocrcamera/LevenshteinStringComparator.java

```diff
@@ -0,0 +1,60 @@
+package unipd.se18.ocrcamera;
+
+import info.debatty.java.stringsimilarity.*;
+
+/**
+ * This class compares strings using weighted levenshtein algorithm
+ * For more informations about the library we are using:
+ * https://github.com/tdebatty/java-string-similarity
+ * @author Francesco Pham
+ */
+public class LevenshteinStringComparator extends WeightedLevenshtein{
+
+    public LevenshteinStringComparator(){
+        super(
+            new CharacterSubstitutionInterface() {
+                public double cost(char c1, char c2) {
+
+                    // The cost for substituting 't' and 'r' is considered
```

Comment:
Why are you reducing the cost of substituting these chars even if we are not using a keyboard?

Reply:
Sorry that comment was copied from elsewhere. Yes you are right, we could change and add more similar characters to the method.